tests/analysis/test_schemas.py

```
@@ -0,0 +1,43 @@
+
+
+def test_rejects_text_over_max_length():
+    with pytest.raises(ValidationError):
+        SmishingAnalysisRequest(
+            text="가" * (settings.MAX_ANALYSIS_CONTENT_LENGTH + 1)
+        )
+
+
+def test_oversized_text_is_not_leaked_in_validation_error():
+    """hide_input_in_errors=True 로 검증 실패 시 원문(PII)이 에러에 담기지 않아야 한다."""
+    secret_marker = "01012345678-비밀번호-보이스피싱"
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to: "secret_marker"

(S105)